files/en-us/web/css/anchor/index.md

```diff
@@ -157,16 +155,58 @@ This example positions the positioned element's logical block end edge `10px` fr
 
 ### Positioning an element relative to multiple anchors
 
-You can position an element relative to multiple anchors by specifying different `<anchor-element>` values inside different `anchor()` functions on the same element (see the [Element positioned relative to multiple anchors](#element_positioned_relative_to_multiple_anchors) demo below). This could be used to create useful functionality, for example, drag handles at the corners of a positioned element, which can be used to resize it.
+You can position an element relative to multiple anchors by specifying different `<anchor-element>` names inside the `anchor()` function of different inset properties on the same element (see [Element positioned relative to multiple anchors](#element_positioned_relative_to_multiple_anchors) below). This can be used to create useful functionality such as drag handles at the corners of a positioned element that can be used to resize it.
 
-While a positioned element can be positioned relative to more than one anchor element, it is only ever associated with the single anchor defined via its [`position-anchor`](/en-US/docs/Web/CSS/position-anchor) property (or the [`anchor`](/en-US/docs/Web/HTML/Global_attributes/anchor) HTML attribute). This is the anchor the element will scroll with when the page scrolls.
+While a positioned element can be positioned relative to more than one anchor element, it is only ever associated with the single anchor defined via its [`position-anchor`](/en-US/docs/Web/CSS/position-anchor) property (or the [`anchor`](/en-US/docs/Web/HTML/Global_attributes/anchor) HTML attribute). This is the anchor the element will scroll with when the page scrolls and controls when the element is [conditionally hidden](/en-US/docs/Web/CSS/CSS_anchor_positioning/Try_options_hiding#conditionally_hiding_anchor-positioned_elements).
 
-### Formal syntax
+## Formal syntax
 
 {{csssyntax}}
 
 ## Examples
 
+### Common usage
+
+In this example, the `anchor()` function is used to set height of an anchor positioned element to the height of its anchor by setting the bottom and top edges to the bottom and top edges of the anchor. The `anchor()` function within a `calc()` function is used to offset the anchor positioned element from its anchor.
```

Comment:
added a basic example to show actual usage as a first example.
https://codepen.io/estelle/pen/OJYWmQL

i can't get my dev env to render this content (I think the branch is too far off from my main, that dependencies are different), so did not test in this page, but the codepen works.

i didn't hide the colors or borders and it's just a few extra lines, but feel free to if you prefer.